cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -113,10 +113,10 @@ elseif(OMR_OS_LINUX)
 		-qxflag=selinux
 	)
 elseif(OMR_OS_ZOS)
-	set(OMR_ZOS_COMPILE_ARCHITECTURE "7" CACHE STRING "z/OS compile machine architecture")
-	set(OMR_ZOS_COMPILE_TARGET "zOSV1R13" CACHE STRING "z/OS compile target operating system")
+	set(OMR_ZOS_COMPILE_ARCHITECTURE "10" CACHE STRING "z/OS compile machine architecture")
+	set(OMR_ZOS_COMPILE_TARGET "ZOSV2R3" CACHE STRING "z/OS compile target operating system")
 	set(OMR_ZOS_COMPILE_TUNE "10" CACHE STRING "z/OS compile machine architecture tuning")
```

Comment:
If updating target OS to V2R3 implicitly implies we are compiling wtih XL C/C++ 2.3 or newer, we should update the [TUNE](https://www.ibm.com/docs/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.cbcux01/tunecop.htm#tunecop) setting to `12`, which corresponds with IBM z14, as much users would be on the most recent generations of the hardware.  

`TUNE(12)` is supported on z/OS V2R3 XL C/C++: https://www.ibm.com/docs/en/zos/2.3.0?topic=cc-whats-new-zos-v2r3-xl

The corresponding change will need to be made in the other files too.

Reply:
@joransiu should OpenJ9 also be making this change to TUNE from 10 to 12?

Reply:
Created https://github.com/eclipse-openj9/openj9/pull/14668